types/types.ts

```diff
@@ -171,3 +171,67 @@ export interface ToastOptions {
     | "bottom-center"
     | "bottom-right";
 }
+
+// Annotated Collections Types
+export interface AnnotatedCollection {
+  id: string;
+  name: string;
+  description?: string;
+  collection_type: "incident" | "wildlife_sighting" | "research_plot";
+  status: "active" | "archived" | "draft";
+  created_by: string;
+  created_at: string;
+  updated_at: string;
+  is_active: boolean;
+  metadata: Record<string, string | number | boolean>;
+}
+
+export interface Incident {
+  collection_id: string;
+  incident_type?: string;
```

Comment:
I think we want to make this a "select one" field with options like "mining", "logging", "petroleum drilling", "poaching", "land invasions", "wildlife trafficking". Which then raises the typical survey design question: what happens if it's not one of the above? Do we support "other" and then another field `incident_type_other` (String)?